src/ConnectKafka.cpp

```diff
@@ -30,17 +47,25 @@ std::unique_ptr<RdKafka::Metadata> ConnectKafka::queryMetadata() {
   RdKafka::Metadata *metadataRawPtr(nullptr);
   Consumer->metadata(true, nullptr, &metadataRawPtr, 1000);
   std::unique_ptr<RdKafka::Metadata> metadata(metadataRawPtr);
-  if (!metadata) {
-    throw std::runtime_error("Failed to query metadata from broker");
+  try {
+    if (!metadata) {
+      throw std::runtime_error("Failed to query metadata from broker");
+    }
+  } catch (std::runtime_error &E) {
```

Comment:
You're not doing anything different with `runtime_error` to any other `exception`, so there is no point having the `catch (std::runtime_error &E)`